modules/ROOT/pages/index.adoc

```diff
@@ -6,20 +6,20 @@ endif::[]
 Run and manage Mule apps on your local servers following your security policies by using Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). Anypoint Platform PCE offers local MuleSoft control plane functionality, with no external system interactions or internet connectivity required, maintaining all data storage, processing, and transmission.  
 Anypoint Platform PCE uses Kubernetes for built-in high availability and scalability. While you can install it without deep Kubernetes knowledge, understanding these tools is essential for management and troubleshooting.
 
-Anypoint Platform PCE version 4.0 runs on enterprise-ready Kubernetes Red Hat OpenShift, Amazon EKS Anywhere (EKS-A), and Rancher Kubernetes Engine (RKE2). Additionally, it supports Java 17, in addition to Java 8 and 11.
+Anypoint Platform PCE version 4.x runs on enterprise-ready Kubernetes Red Hat OpenShift, Amazon EKS Anywhere (EKS-A), and Rancher Kubernetes Engine (RKE2). Additionally, it supports Java 17, in addition to Java 8 and 11.
 
 For information about a previous version of Anypoint Platform PCE, select the version in the documentation sidebar.
 
 == Before You Begin
 
-=== Upgrade to Anypoint Platform PCE version 4.0
-If you are upgrading from version 3.2.3 of Anypoint Platform PCE to version 4.0, see xref:upgrade.adoc[Upgrading Anypoint Platform PCE 3.2.3 to 4.0].
+=== Upgrade to Anypoint Platform PCE version 4.X
+If you are upgrading from version 3.2.3 of Anypoint Platform PCE to version 4.X, see xref:upgrade.adoc[Upgrading Anypoint Platform PCE 3.2.3 to 4.x].
 
 [NOTE]
-If you're using a version of Anypoint Platform PCE earlier than 3.2.3, upgrade to version 3.2.3 first. Then proceed with upgrading to version 4.0.
+If you're using a version of Anypoint Platform PCE earlier than 3.2.3, upgrade to version 3.2.3 first. Then proceed with upgrading to version 4.x.
```

Comment:
Also @HONTAKYU is this NOTE relevant now? Or should this be updated as well?

Reply:
This should be fine, no need for update